tests/data/deltaorion_deltaorion.json

```diff
@@ -0,0 +1,437 @@
+{
+    "os": {
+        "discovery": {
+            "devices": [
+                {
+                    "sysName": "<private>",
+                    "sysObjectID": ".1.3.6.1.4.1.20246",
+                    "sysDescr": "Orion Power System Controller. Delta Electronics (Switzerland) AG.",
+                    "sysContact": "<private>",
+                    "version": "B8.35Build03",
+                    "hardware": null,
```

Comment:
Is there any Oid for hardware?

Reply:
There is no hardware discription when I walk the device.  Here is what is known...
dcSiteName.0 = XXX
dcSystemName.0 = XXX
dcSystemDateTime.0 = 2021-04-12T09:06:52
dcSoftwareVersion.0 = B8.35Build03
dcSystemInfo.5.0 = 1